app/ui/Home.py

```diff
@@ -1,46 +1,206 @@
 import streamlit as st
-import config
 import sthelp
-from modules import add_custom_modules
+from connection import execute_with_cache
+import datetime
 
-if not add_custom_modules():
-    st.warning("Unable to load OpsCenter modules.")
-import permissions
 
-sthelp.chrome()
-permissions.setup_permissions()
+st.set_page_config(layout="wide", page_title="Sundeck Opscenter", page_icon=":pilot:")
 
-st.markdown(
-    """
-# Welcome To Sundeck OpsCenter
-
-## Overview
-OpsCenter is a community-driven, freely available Github project. It provides a set of tools to help you manage your
-Snowflake account. If you want to know more about the capabilities, check them out
-[here](https://sundeck.io/community/opscenter).
-
-Check out the items on the sidebar.
-"""
-)
-if config.has_tenant_url():
-    tenant_url = config.get_tenant_url()
-    st.markdown(
-        f"""
-            To explore the Sundeck account, right-click on the link below and choose "Open link in new tab/window."
 
-            [Go to my Sundeck account]({tenant_url})
-        """
+def timedelta_to_human_readable(delta: datetime.timedelta) -> str:
+    if delta.total_seconds() < 0:
+        return None
+    # Break down the timedelta into days, hours, minutes, and seconds
+    days, seconds = delta.days, delta.seconds
+    hours = seconds // 3600
+    minutes = (seconds % 3600) // 60
+    seconds = seconds % 60
+
+    # Build a human-readable string
+    parts = []
+    if days > 0:
+        parts.append(f"{days} day{'s' if days > 1 else ''}")
+    if hours > 0:
+        parts.append(f"{hours} hour{'s' if hours > 1 else ''}")
+    if minutes > 0:
+        parts.append(f"{minutes} minute{'s' if minutes > 1 else ''}")
+    if seconds > 0 or not parts:
+        parts.append(f"{seconds} second{'s' if seconds != 1 else ''}")
+
+    return ", ".join(parts)
+
+
+def get(query, fmt):
+    df = execute_with_cache(query)
+    if df is None:
+        return fmt(None)
+    if df.empty:
+        return fmt(None)
+    util = df.iloc[0, 0]
+    if util is None:
+        return fmt(None)
+    return fmt(util)
+
+
+def to_str(end_time, start_time):
```

Comment:
Please add pydoc for this method since it returns two values.

Reply:
done